### ISSUE TITLE
Make it more clear that tabs were forgotten in background (suggestion)

### ISSUE THREAD

**Issue by vasimi** [OnionBrowser/iOS-OnionBrowser#309]
I didn't use onion browser for a while and I was pretty confused that all my tabs disappeared when I locked my phone. I thought this was a bug and then I found out I can change this behavior in settings.

Wouldn't it be nice if the user was somehow informed that his tabs were forgotten in the background and he can change this behavior in settings?

**Comment by tladesignz** [comment]
Yeah, I know. This is suboptimal. I guess we should just change the default behaviour in a version 3.
Keeping this in evidence.

**Comment by tladesignz** [comment]
Sorry, this didn't make it in 3.0.0.

@cstiens, what could we do to inform the user without changing this (very safe) default?

**Comment by pubgmubashir0-cpu** [comment]
Hi
